README.md

```diff
@@ -1,23 +1,7 @@
-[![Ohloh SCM on OpenHub](https://www.openhub.net/p/ohloh_scm/widgets/project_partner_badge.gif)](https://www.openhub.net/p/ohloh_scm) [![Build Status](https://travis-ci.org/blackducksoftware/ohloh_scm.svg?branch=master)](https://travis-ci.org/blackducksoftware/ohloh_scm)
-
 # Ohloh SCM
 
 The OpenHub source control management library
 
-This program is free software; you can redistribute it and/or modify
-it under the terms of the GNU General Public License Version 2 as
-published by the Free Software Foundation.
-
-Ohcount is specifically licensed under GPL v2.0, and no later version.
-
-This program is distributed in the hope that it will be useful,
-but WITHOUT ANY WARRANTY; without even the implied warranty of
-MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-GNU General Public License for more details.
-
-You should have received a copy of the GNU General Public License
-along with this program.  If not, see <http://www.gnu.org/licenses/>.
-
```

Comment:
I think we need to leave the license information, too

Reply:
We have a LICENSE file which seems like the standard pattern on other repositories on Github. Github also tracks license information from the LICENSE file.